### ISSUE TITLE
Review VAFSC Accessibility Audit Findings

### ISSUE THREAD

**Issue by taylorkaren** [department-of-veterans-affairs/va-forms-system-core#470]
## Task Description
- Review the findings from the VAFSC Accessibility Audit with the audit team (meeting scheduled for 8/11/22); findings will be documented via tickets in the va-forms-system-core repo.
- Do some initial prioritization of issues to address by ordering the tickets in the Accessibility Issues pipeline; in general, higher severity issues should be prioritized first.

## Considerations
Document that will be used to track the audit progress as DK, Angela, Tiffany, and Sarah work on it: https://docs.google.com/spreadsheets/d/1BHRQY6Ddxd9rFLHSYjSdvabp_Uk-mz1j-JeLq6jTT3U/edit#gid=1115838130. 

## Acceptance Criteria
- [x] Forms Library Team members (Micah, Karen, Anish) have met with the accessibility audit team to review and discuss their findings.
- [x] Initial prioritization of findings tickets has been done in the Accessibility Issues pipeline on the Forms Library Team ZenHub board.

**Comment by taylorkaren** [comment]
Notes from the 8/11/22 preliminary findings check in call: [https://app.zenhub.com/files/375815514/e19a1791-9116-480b-9a9f-6c5b1c2643a2/download](https://app.zenhub.com/files/375815514/e19a1791-9116-480b-9a9f-6c5b1c2643a2/download).

Next meeting: Thursday, August 18th. We'll discuss the issues entered into GitHub that have [Discuss} in the title, then move on to questions that we have about any of the other entered issues.

**Comment by taylorkaren** [comment]
Notes from the 8/18/22 issue review and discussion call: [https://app.zenhub.com/files/375815514/8ec37e9b-eef9-4858-a38b-57f6ea6dec1c/download](https://app.zenhub.com/files/375815514/8ec37e9b-eef9-4858-a38b-57f6ea6dec1c/download).
